activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -1371,6 +1365,19 @@ def preprocess_order_args(order_args)
         end.flatten!
       end
 
+      def sanitize_order_arguments(order_args)
+        order_args.reject!(&:blank?)
+        order_args.map! do |arg|
+          klass.sanitize_sql_for_order(arg)
+        end
+        order_args.flatten
```

Comment:
```suggestion
        order_args.flatten!
        order_args
```

We are already mutating the object in the first line of this method, do we should not duplicate it in the last call.

Reply:
Done 👍